src/benchmarkstt/metrics/cli.py

```diff
@@ -0,0 +1,44 @@
+from benchmarkstt.metrics.core import WER
+from benchmarkstt.input import core
+import argparse
+
+
+def argparser(parser: argparse.ArgumentParser):
+    # steps: input normalize[pre?] segmentation normalize[post?] compare
+    parser.add_argument('-r', '--reference', required=True,
+                        help='The file to use as reference')
+    parser.add_argument('-h', '--hypothesis', required=True,
```

Comment:
I've made the decision to not provide a short help syntax, but only support the `--help` argument. It makes the short argument for hypothesis a bit less confusing imho. Plenty of command line programs only support `--help` (eg. git, mysql, docker [deprecated], etc.) so this is in line with common usage of command line tools and avoids having 'longer' short arguments names while completely unnecessary because `-h` would be reserved to the help funciton...

Reply:
Sounds reasonable. Unless @danielthepope object I'm happy with `-h`